doc/lua_api.txt

```diff
@@ -3702,6 +3702,12 @@ Call these functions only at load time!
     * Called when an item is eaten, by `minetest.item_eat`
     * Return `true` or `itemstack` to cancel the default item eat response
       (i.e.: hp increase).
+* `minetest.register_on_item_pickup(func(luaentity, picker, time_from_last_punch, tool_capabilities, dir))`
+    * Called when item is picked up by `picker`.
+    * Function is added to `minetest.registered_on_item_pickups`
+    * Newest functions are called first.
```

Comment:
I might be wrong, but aren't the oldest ones called first? The for loop seems to iterate from first to the last index:

`for i = 1, #core.registered_on_item_pickups do`

Reply:
You're right. >_<

Reply:
Or the callback registration function probably inserts the newer methods first. I guess we'll have to check.

Reply:
Tested. Oldest called first is correct for on_pickup.
`register_on_player_receive_fields` is out of scope.

Reply:
Oh, ok then. :)

Reply:
Btw. I saw now that `register_on_player_receive_fields` uses `make_registration_reverse`, ergo it's documented correctly.